.github/workflows/publish-dev.yml

```
@@ -21,6 +21,11 @@ jobs:
         # Remove --no-optional after Node 16 upgrade
         run: npm ci --ignore-scripts --no-optional
 
+      - name: Deprecate old versions
+        run: npm deprecate discord.js@"~13.0.0-dev" "No longer supported" || true
```

Can't something like this be done, so it auto deprecates any valid dev version from the package.json?

```suggestion
        run: npm deprecate discord.js@"~$(jq --raw-output '.version' package.json)" "No longer supported" || true
```

---

I thought this wasn’t gonna be used after v13 but if it will then yeah I think that can work

---

Oh, gotcha

---

For now, yes.